utils/get-grpc.ps1

```diff
@@ -0,0 +1,61 @@
+##===- utils/get-grpc.ps1 - Install gRPC (for ESI runtime) ---*- Script -*-===##
+# 
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# 
+##===----------------------------------------------------------------------===##
+#
+#
+##===----------------------------------------------------------------------===##
+
+$ErrorActionPreference = "Stop"
+
+# Create the 'ext' directory relative to the script location
+$scriptDir = Split-Path -Parent $MyInvocation.MyCommand.Path
+$extDir = Join-Path $scriptDir "..\ext"
+New-Item -ItemType Directory -Path $extDir -Force | Out-Null
+
+Write-Output $extDir
+Set-Location $extDir
+
+$vcpkgRoot = $env:VCPKG_ROOT
+
+# Check if VCPKG_ROOT is set
+if (-Not $vcpkgRoot) {
+    Write-Output "VCPKG_ROOT is not set..."
+    # Install vcpkg if vcpkg executable isn't available in path
+    if (Test-Path -Path ".\vcpkg.exe") {
```

Comment:
Have you tested this? I don't think it'll work if the `.\` is included and I don't think test test-path tests whether an executable exists in the PATH. (Though I don't know anything about pwsh)